indigo/docs/guides/howto-hello-indigo-tyrian.md

```diff
@@ -0,0 +1,418 @@
+---
+id: howto-responsive-ui-with-tyrian
+title: How to make a custom entity
+---
+
+## Responsive UI Using Tyrian
+
+There tend to be two primary types of UI in games: In-game UI (such as health over a
+player), and overlay (such as a HUD or dialog box). In-game UI tends not to
+worry too much about the screen size, and is dealt with primarily by the
+magnification configuration within Indigo. But what about overlays and HUD's?
+These tend to scale with the size of the screen and the position of such
+elements may change depending on the capabilities of the display. Many game
+engines provide a way to create overlay UI elements that scale in some way to
+the space they're in using a separate UI system. In Indigo we're exporting to
+JavaScript so it makes sense for us to use other web technologies for a problem
+that is already, ultimately, solved.
```

Comment:
> In Indigo we're exporting to JavaScript so it makes sense for us to use other web technologies for a problem
that is already, ultimately, solved.
Sounds little like a recommended solution, I'd rather it read in such a way that we're showing people their options, maybe:

"In Indigo we're exporting to JavaScript and a browser (even in Electron), so depending on how we want the UI to look, one option is to employ more standard web technologies to do this work for us, i.e. HTML and CSS."

Or something?

Reply:
Makes sense